packages/tools/devtools/devtools-browser-extension/e2e-tests/chromebrowserdevtool.test.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { globals } from "../jest.config";
+import { Page, Browser } from "puppeteer";
+
+describe("chrome browser...", () => {
+	let browser: Browser;
+	let page: Page;
+	const puppeteer = require("puppeteer");
+
+	beforeAll(async () => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		// Wait for the page to load first before running any tests
+		// so this time isn't attributed to the first test
+		await page.goto(globals.PATH, { waitUntil: "domcontentloaded" });
+	});
+
+	afterAll(async () => {
+		// Close the browser
+		await browser.close();
+	});
+
+	it("Verify textarea is created", async () => {
+		document.getElementsByClassName("text-area");
+		// expect(divs.item.length).toBe(1);
```

Comment:
Let's try to be more specific here. When rendering, give the components an `id`, and when looking for them here do we can do it by id, to avoid any potential of finding something that matches the query but isn't actually the thing we're expecting. Also we'll need that `expect()` uncommented 😄 .

Reply:
Modifying...but the id still undefined!!Ah!!